RileyLinkKit/PumpState.swift

```diff
@@ -23,8 +23,10 @@ public class PumpState {
     /// If the value type does not conform to AnyObject, a raw representation will be provided instead.
     public static let ValueChangeOldKey = "com.rileylink.RileyLinkKit.PumpState.ValueChangeOldKey"
     
-    public let pumpID: String
+    public var scanFrequencies = [916.45, 916.50, 916.55, 916.60, 916.65, 916.70, 916.75, 916.80]
```

Comment:
I worry about any public mutable array anywhere :) But let's see where this goes.